hangman_colorize.rb

```diff
@@ -0,0 +1,194 @@
+require "Colorize"

+

+class Hangman


```

Comment:
Be wary of using too-much white space between lines. Keep it to one empty line max